cdn_lambda/functions/map_to_s3/map_to_s3.py

```diff
@@ -31,6 +31,12 @@ def db_client(self):
 
         return self._db_client
 
+    def uri_alias(self, uri):
+        for key, value in self.conf["uri_aliases"].items():
+            if key in uri:
```

Comment:
It shouldn't be key "in" uri, it should be if the uri path startswith key, right?  The key could appear somewhere other than the beginning, in which case it'd be wrong to replace it.

It's not a negligible risk, for example there is a project associated with Red Hat called "origin" (openshift origin) so it's not that unlikely that strings such as /content/origin/ or /origin/rpm/ could appear somewhere in the URL other than at the beginning.

Reply:
Ah, I didn't think that was a realistic scenario. I'll make the change but it has consequences as you've already noticed.